### PR TITLE
Updated RangeFinder/Lidar doc relatively to semi-transparent objects

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked with omichel/webots#3799

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: reference/lidar.md [hunk 1]
```diff
@@ -37,9 +37,7 @@ field of view is imposed by the size (width and height) of the image (because of
 the constraint of square pixels) and not in the case of the [Lidar](#lidar) node
 where lines of pixels (laser scan) are extracted from the depth buffer.
 
-Lidar cannot see semi-transparent objects. An object can be semi-transparent
-either if its texture has an alpha channel, or if its
-[Material](material.md).`transparency` field is not equal to 1.
+Lidar cannot see semi-transparent objects as if they were not transparent. An object can be semi-transparent either if its texture has an alpha channel, or if its [Material](material.md).`transparency` field is not equal to 1.
```
Comment (stefaniapedrazzi) [inline reference/lidar.md]: `cannot` should be removed

Other typos not related to this PR that applies to Range-finder section as well:
* `see` should be `sees` (I would maybe use the verb `detects`)
* there is an additional `.` just before transparency

**Comment by DavidMansolino** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**